exporter.go

```diff
@@ -170,6 +170,18 @@ var (
 		prometheus.BuildFQName(namespace, "cluster_health", "status_is_green"),
 		"Whether all primary and replica shards are allocated.",
 		[]string{"cluster"}, nil)
+	clusterHealthStatusDesc = prometheus.NewDesc(
```

Comment:
This metric should be more usefull than status_is_green, statis_is_yellow and status_is_read